applications/nrf5340_audio/unicast_client/README.rst

```diff
@@ -0,0 +1,151 @@
+.. _nrf53_audio_unicast_client_app:
+
+nRF5340 Audio: Unicast client
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF5340 Audio unicast client application implements the :ref:`CIS gateway mode <nrf53_audio_app_overview>`.
+
+In this mode, one Connected Isochronous Group (CIG) can be used with two Connected Isochronous Streams (CIS).
+Transmitting unidirectional or transceiving bidirectional audio happens using CIG and CIS.
+
+The following limitations apply to this application:
+
+* One CIG with two CIS.
+* Audio input: USB or I2S (Line in or using Pulse Density Modulation).
+* Audio output: USB or I2S/Analog headset output.
+* Configuration: 16 bit, several bit rates ranging from 32 kbps to 124 kbps.
+
+.. _nrf53_audio_unicast_client_app_requirements:
+
+Requirements
+************
+
+The application shares the :ref:`requirements common to all nRF5340 Audio application <nrf53_audio_app_requirements>`.
+
+.. _nrf53_audio_unicast_client_app_ui:
+
+User interface
+**************
+
+Most of the user interface mappings are common across all nRF5340 Audio applications.
+See the :ref:`nrf53_audio_app_ui` page for detailed overview.
+
+This application uses specific mapping for the following user interface elements:
+
+* Long-pressed on the unicast client device during startup:
+
+  * **BTN5** - Clears the previously stored bonding information.
+
+* Pressed on the unicast client device during playback:
```

Comment:
Should we have this info here as this is well documented in user_interface.rst?

Reply:
We can discuss this.

Reply:
Discussed and kept.